src/lib/eina/eina_thread.c

```diff
@@ -23,129 +23,56 @@
 #include <stdlib.h>
 
 #include "eina_config.h"
-#include "eina_lock.h" /* it will include pthread.h with proper flags */
```

Comment:
In a case of a Linux compiling, those still make sense this order? I mean, is it still relevant, if so, maybe it is a good idea to let this comment on the C file. Or even change it a bit, like, "in case of a POSIX build, it will include pthread.h with proper flags"

Reply:
maybe put it in eina_thread_posix.h?

Reply:
Oh, I've just had a little deeper look at that. As both implementations need `eina_lock.h` it makes more sense here. Unless you want to explicitly include `eina_lock.h` on both `eina_lock_{posix,win32}.h` and put the comment on the `posix.h` only.

Reply:
i'll let it on eina_thread.c with "// in case of a posix build ..." is it ok?